app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+    :recoverable, :rememberable, :validatable
+  has_many :memberships, dependent: :destroy
+  has_many :competitions, through: :leagues
+  has_many :predictions, dependent: :destroy
+  has_many :matches, through: :predictions
+  validates :name, presence: true
+  validates :timezone, presence: true
```

Comment:
I would remove the presence validations on the `name` and `timezone` here. We can always force them to set a name before being able to access any other page in the FE, but I would stick to the strict minimum inputs to signup, especially since many will probably signup from a mobile device.

If they abandoned at the name step, we can always send them a reminder that their friends are waiting on the app as long as we have the email.

We can assign users on a default timezone as well.